frank/radial_fitters.py

```diff
@@ -693,7 +699,7 @@ def __init__(self, Rmax, N, geometry, nu=0, block_data=True,
                  block_size=10 ** 5, alpha=1.05, p_0=None, weights_smooth=1e-4,
                  tol=1e-3, method='Normal', I_scale=1e5, max_iter=2000, check_qbounds=True,
                  store_iteration_diagnostics=False, assume_optically_thick=True,
-                 scale_height=None, verbose=True):
+                 scale_height=None, verbose=True, convergence_failure='raise'):
```

Comment:
Better if the default is 'warn', since for most (non-debris) fits the fit is only marginally changing at large iteration, but set it as `raise` in `FrankDebrisFitter`?